docs/api/verify-service-provider-api.swagger-v2.yml

```diff
@@ -0,0 +1,293 @@
+                    code: 400
+                    message: Some error message
+        '422':
+          description: An error due to a JSON request in an invalid format (e.g. missing mandatory parameters).
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMessage'
+            errorMessage:
+              examples:
+                response:
+                  value:
+                    code: 422
+                    message: Some error message
+        '500':
+          description: An error due to an internal server error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorMessage'
+            errorMessage:
+              examples:
+                response:
+                  value:
+                    code: 500
+                    message: Some error message
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TranslateSamlResponseBody'
+        description: An object containing the SAML response returned by the Verify service.
+        required: true
+components:
+  schemas:
+    RequestGenerationBody:
+      type: object
+      properties:
+        levelOfAssurance:
+          $ref: '#/components/schemas/RequiredLevelOfAssurance'
+        entityId:
+          $ref: '#/components/schemas/EntityId'
+    RequestResponseBody:
+      type: object
+      required:
+        - samlRequest
+        - requestId
+        - ssoLocation
+      properties:
+        samlRequest:
+          description: SAML Authn Request as a base64 string.
+          type: string
+          format: byte
+        requestId:
+          description: A token that identifies the Authn Request. This is used later to verify that the Authn Request and SAML Response have passed through the same browser.
+          type: string
+          format: byte
+        ssoLocation:
+          description: The url for Verify Hub SSO. The entrypoint for SAML authentication flow.
+          type: string
+          format: url
+    TranslateSamlResponseBody:
+      type: object
+      required:
+        - samlResponse
+        - requestId
+        - levelOfAssurance
+      properties:
+        samlResponse:
+          description: A SAML Response as a base64 string.
+          type: string
+          format: byte
+        requestId:
+          description:  A token that was generated for the original SAML Authn Request. This is used to verify that the Authn Request and SAML Response have passed  through the same browser.
+          type: string
+          format: byte
+        levelOfAssurance:
+          $ref: '#/components/schemas/RequiredLevelOfAssurance'
+        entityId:
+          $ref: '#/components/schemas/EntityId'
+    TranslatedResponseBody:
+      type: object
+      required:
+        - scenario
+        - pid
+        - attributes
```

Comment:
Thinking about this... if the scenario is anything other than IDENTITY_VERIFIED, we won't have a PID or any attributes. I'll take 'em out.